Dapper NET40/SqlMapper.cs

```diff
@@ -3095,15 +3092,7 @@ internal static IList<LiteralToken> GetLiteralTokens(string sql)
             if (ctors.Length == 1 && propsArr.Length == (ctorParams = ctors[0].GetParameters()).Length)
             {
                 // check if reflection was kind enough to put everything in the right order for us
-                bool ok = true;
-                for (int i = 0; i < propsArr.Length; i++)

```

Comment:
replacing a perfectly good foreach loop with LINQ is *not* code improvement, and adds a whole host of additional overhead to the method; will reject